governance/votes/templates/Executive vote - September 13, 2024.md

```diff
@@ -0,0 +1,222 @@
+- `usds`: [0xdC035D45d973E3EC169d2276DDab16f1e407384F](https://etherscan.io/address/0xdC035D45d973E3EC169d2276DDab16f1e407384F)
+- `splitter`: [0xBF7111F13386d23cb2Fba5A538107A73f6872bCF](https://etherscan.io/address/0xBF7111F13386d23cb2Fba5A538107A73f6872bCF)
+- `prevChainlogKey`: MCD_FLAP
+- `chainlogKey`: MCD_FLAP
+
+##### Oracle Initialization
+
+The Oracle will be initialized by calling the [FlapperInit.initOracleWrapper ](https://github.com/makerdao/dss-flappers/blob/bddeb5ac09a5ea6c5b4bc77cec1bd5b05014f1fd/deploy/FlapperInit.sol#L141) function with the following parameters:
+
+- `wrapper_`: [0x38e8c1D443f546Dc014D7756ec63116161CB7B25](https://etherscan.io/address/0x38e8c1D443f546Dc014D7756ec63116161CB7B25)
+- `divisor`: 24,000
+- `clKey`: FLAP_SKY_ORACLE
+
+### dssVestableMint Setup
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Poll 1132]($TBD)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/1)
+
+If this executive proposal passes, then DssVestMintable for SKY will be set up through the following actions:
+
+- **Authorize DssVestMintable on SKY by calling `DssExecLib.authorize` with the following parameters:**
+  - `_base`: [0x56072C95FAA701256059aa122697B133aDEd9279](https://etherscan.io/address/0x56072C95FAA701256059aa122697B133aDEd9279)
+  - `_ward`: [0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9](https://etherscan.io/address/0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9)
+- **Set DssVestMintable max rate (`cap`) by calling `DssExecLib.setValue` with the following parameters:**
+  - `_base`: [0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9](https://etherscan.io/address/0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9)
+  - `_what`: "cap"
+  - `_amt`: 799,999,999.999999999985808000 Sky per year (800M * WAD / 365 days )
+- **Add DssVestMintable to Chainlog by calling `DssExecLib.setChangelogAddress` with the following parameters:**
+  - `_key`: MCD_VEST_SKY
+  - `_val`: [0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9](https://etherscan.io/address/0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9)
+
+### USDS->SKY Farming Setup
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Poll 1132]($TBD)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/1)
+
+If this executive proposal passes, then the USDS->SKY farming setup will be initialized through the following actions:
+
+- **Initialize USDS -> SKY rewards by calling [UsdsSkyFarmingInit.init](https://github.com/makerdao/endgame-toolkit/blob/14268515aa729a588096f0d579ea38bde3e9ba2f/script/dependencies/phase-1b/UsdsSkyFarmingInit.sol#L42) with the following parameters:**
```

Comment:
"USDS -> SKY" or "USDS->SKY"? Either is fine, but it should be consistent throughout the document.